osidb/tests/endpoints/test_endpoints.py

```diff
@@ -382,10 +382,12 @@ def test_custom_exception_serialization(self):
         assert response.status_code == 409
         assert response.json()["detail"] == "This was a big failure"
 
+
```

Comment:
nitpick: these formatting changes should be part of the previous commit (no need to change it)